### ISSUE TITLE
Lighting flickering when player moving and looking around

### ISSUE THREAD

**Issue by davetgc** [TheGameCreators/GameGuruRepo#5086]
The video pretty much explains it all. A light, possibly the sun is being weirdly culled from certain perspectives. I first noticed this yesterday after adding a new light, since then deleting the lights makes no difference.

https://github.com/TheGameCreators/GameGuruRepo/assets/99803873/b81f89bf-bb29-4e8d-a0c2-9867761e166e

**Comment by LeeBamberTGC** [comment]
@davetgc What happens if you set the shadow counts to 16 each:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/06261ee2-4e90-42ab-a678-33ba9f4cabce)

**Comment by davetgc** [comment]
Just tested it, it's the same, still appearing to be culled.

**Comment by LeeBamberTGC** [comment]
@davetgc Any chance you can make a standalone of it and email to me?  I can then run in debug and see what might be changing, thanks!

**Comment by davetgc** [comment]
@LeeBamberTGC - I'll do that for you tomorrow morning.

**Comment by davetgc** [comment]
Hi Lee,

I've built a standalone, and the issue isn't present there, just in test game. Do you still want the build? Not sure it'll be much use.

**Comment by LeeBamberTGC** [comment]
@davetgc No worries, send the standalone, and I can revert it back to something I can load in the test level to see the issue, no worries.

**Comment by davetgc** [comment]
@LeeBamberTGC - I've sent you a link via email. Let me know if you need more information.

**Comment by LeeBamberTGC** [comment]
@davetgc Thanks for the email, as you say, the standalone does not show the issue. I did find a way to create a similiar flicker, related to how many point lights are in close enough proximity and casting shadows:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/06bdf6db-7741-4e40-afe5-0d03e41704b3)

When I reduced it from 12 to 8 in the switch escape demo, I could get it to flicker in room 1.
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/ef17aa75-eaaf-4cf3-a516-841907bb1a7f)
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/b8a34071-9f5f-48fd-a6b7-eb72e8ec9cc9)

As a test remove lots of point lights in the area around the flicker and see if that makes it go away. It could be the scene is asking for more than 16 point lights.
